src/client/dfs/SConscript

```diff
@@ -1,5 +1,7 @@
 """Build FUSE client"""
 import daos_build
+import os.path
+from os import path
```

Comment:
Isn't this effectively double-importing `os.path` into two different namespaces?

Reply:
right!